### ISSUE TITLE
Not working at all

### ISSUE THREAD

**Issue by molitar** [YePpHa/YouTubeCenter#2228]
There is nowhere to configure the Youtube Center anymore.  I only have the usual Youtube options.  Totally broken in Firefox.

**Comment by tooobe** [comment]
Totally broken in Chrome as well. Both stable and beta channel.

**Comment by molitar** [comment]
I downloaded the script and using greasemonkey works fine there.  The plugins are broken.

**Comment by tooobe** [comment]
@molitar I'm using Tampermonkey and it's not working...

**Comment by Wishbringer** [comment]
Well, I'm using the greasemonkey version and its not working anymore. Stopped working a little over a week ago. Either its my Firefox Nightly build (doubtful) or Youtube changed something (very likely).

**Comment by sup2069** [comment]
Nope, its firefox! After version 43.0.2 the add-on is broken. Reverting back fixed to 43.0.2 fixed this for me months ago! The only downside is security vulnerabilities and what not, but I cant live without this add-on, ill take the risk.

**Comment by Wishbringer** [comment]
Sup2069: Well, I'm sorry but you're talking about the plugin version not the Greasemonkey script (plugin wouldn't save settings). The Greasemonkey script was working just fine a few weeks ago with Firefox Nightly.

**Comment by sup2069** [comment]
Yes the plugin version. Sorry I misread your post earlier!

**Comment by Nicholas-Steel** [comment]
The YouTube Center Developer Build 538 combined with Greasemonkey 3.7 works fine in Firefox 45.0.2 under Windows 10 x64.

**Comment by jcpro** [comment]
@Nicholas-Steel I'm running GM 3.7, FF 45.0.2, YTC Dec Buil 538 and it's still broken.  Can you confirm where you got Build 538 from?

**Comment by Wishbringer** [comment]
Also using GM 3.7 with Build 538 (from dist on here) but with FF 48. Not working for me, could be my Nightly, then again others are having this problem.

**Comment by Nicholas-Steel** [comment]
https://github.com/YePpHa/YouTubeCenter/raw/master/dist/YouTubeCenter.user.js via https://github.com/YePpHa/YouTubeCenter/wiki/Developer-Version

**Comment by Wishbringer** [comment]
Weirdly, reinstalling the script fixed my problem. :D
